Tools/scripts/build_ci.sh

```diff
@@ -46,7 +53,7 @@ for t in $CI_BUILD_TARGET; do
         (cd modules/mavlink/pymavlink && python setup.py build install --user)
         unset BUILDROOT
         echo "Running SITL QuadCopter test"
-        Tools/autotest/autotest.py build.ArduCopter fly.ArduCopter
+        Tools/autotest/autotest.py build.ArduCopter fly.ArduCopter $autotest_args
```

Comment:
Sorry I forgot about it. The variable has spaces, making it `"$autotest_args"` **should** work.